doc/migration.md

```diff
@@ -0,0 +1,83 @@
+# Hypersync SDK Migration Guide
+
+## Version 2.X to Version 3.0
+
+Version 3.0 of the Hypersync SDK includes a number of new capabilities that
+require small changes to Hypersync apps that use version 2.1 or earlier.
+
+### Package Versions
+
+The Hypersync SDK functionality is distributed via three Node packages:
+
+| Package                      | Version | Description                 |
+| ---------------------------- | ------- | --------------------------- |
+| @hyperproof/hyperysnc-sdk    | 3.0.2   | Core Hypersync SDK          |
+| @hyperproof/hypersync-models | 5.0.0   | Supporting Hypersync models |
+| @hyperproof/integration-sdk  | 1.0.2   | Common integration elements |
+
+All of the functionality needed to develop Hypersync applications is now
+exported from `@hyperproof/hypersync-sdk`.  There is no longer a need to
+include dependencies to `@hyperproof/hypersync-models` or `@hyperproof/integration-sdk`.
+
+The `dependencies` section of your app's `package.json` file should be updated as follows:
+
+```
+  "dependencies": {
+    "@hyperproof/hypersync-sdk": "^3.0.2",
+    ...
+  }
+```
+
+If your code imported classes, types, interfaces, or enums from one of the
+other packages, those imports will need to be updated after making this change.
+
+### Updated Execution Environment
+
+Hypersync apps in Hyperproof EU execute in an updated execution environment. A couple of
```

Comment:
Do we want to mention FR here as well, or too soon?

Reply:
Too soon.  I have a work item to update this with Hyperproof Gov refs once we get there.